emanate/observers.py

```diff
@@ -60,15 +61,19 @@ class RabbitMQPublisher(QueuePublisher):
     By default, messages are published on a headers exchange. The name of the
     exchange is given in the `config` passed to the Publisher's constructor.
 
+    If the 'incessant' flag is set, then the publisher will continuously attempt
+    to try and publish an event regardless of any errors - no new events will be
+    processed until the current event has been successfully published.
```

Comment:
Logic that retries publishing an event after a transient error would be a great addition.  However, that logic needs to handle permanent errors (e.g., ``pika.exceptions.ProtocolSyntaxError``) differently from transient errors (network down).  In addition, even for transient error conditions, there should be a limited number of retries per event.  The approach of retrying indefinitely has too much potential for creating infinite loops, and won't prevent the loss of events, since the in-memory queue that a publisher is processing can quickly become full if the publisher becomes stuck on a single event.